### DISCUSSION TITLE
establish compatibility with Transport-Pipes

### DISCUSSION THREAD

**Discussion by McBesser** [BG-Software-LLC/WildChests#72] [Implemented Feature Requests] [answered]
Would it be possible to make this plugin compatible with the Transport-Pipes plugin? (https://github.com/BlackBeltPanda/Transport-Pipes/releases)

here is the answer to my feature request for the plugin 

> BlackBeltPanda 

> Transport-Pipes handles moving the items from container to container. It's up to other plugins to add support for Transport-Pipes if they're using their own custom containers. We have an API other plugins can use to register TransportPipesContainers and handle the different Duct events: https://github.com/BlackBeltPanda/Transport-Pipes/tree/master/src/main/java/de/robotricker/transportpipes/api

https://github.com/BlackBeltPanda/Transport-Pipes/issues/17

And thx for very nice plugin

**Comment by Skizzles** [comment, marked as answer]
@McBesser This was added by Omer!
